assets/stylesheets/app/_flight_manual.scss

```diff
@@ -481,6 +481,17 @@ input.documents-search-input {
   }
 }
 
+// Adjustments for page-specific links to improve the docs
+$improve-link-separator-spacing: 1em;
```

Comment:
@simurai - I defined a SASS variable, `improve-link-separator-spacing`, which *should* specify the total distance between the improve-link and the article title (if I did my math right). That should make it easier to tweak the style on the fly. But let me know if I went overboard on defining my variable or naming it poorly :innocent: